build.gradle.kts

```diff
@@ -15,3 +16,17 @@ val parameters by tasks.registering {
     description = "Displays build parameters (i.e. -P flags) that can be used to customize the build"
     dependsOn(gradle.includedBuild("build-logic").task(":build-parameters:parameters"))
 }
+
+nmcpAggregation {
+    centralPortal {
+        username = providers.environmentVariable("CENTRAL_PORTAL_USERNAME")
+        password = providers.environmentVariable("CENTRAL_PORTAL_PASSWORD")
+        publishingType = "USER_MANAGED"
+        publicationName = "sigstore protobuf-specs $version"
+    }
+}
+
+dependencies {
+    nmcpAggregation(project(":sigstore-java"))
+    nmcpAggregation(project(":sigstore-maven-plugin"))
```

Comment:
Please add gradle plugin artifacts as well

Reply:
this is creating some issue where it's trying to apply the plugin on itself? We can try to add it in after, but for now, I'd like to get this in so we can publish these (and still publish the other to the portal)

Reply:
It's not exactly set up like that. `publish-to-central` only has the config to publish to central; while signing, etc is all configured in `java-published-library`

We've run into this problem before where the plugin cannot be applied on itself. I'm happy to explore this again though, currently it's blocking with an unclear solution.

Reply:
any update here? I would like to unblock this.